crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/controller/StatelessCMAControllerTest.java

```diff
@@ -0,0 +1,162 @@
+import java.util.stream.Stream;
+
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.DEFINED_PORT;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.httpBasic;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@Import(CrimeMeansAssessmentTestConfiguration.class)
+@SpringBootTest(classes = {CrimeMeansAssessmentApplication.class}, webEnvironment = DEFINED_PORT)
+public class StatelessCMAControllerTest {
+    private static final String CLIENT_SECRET = "secret";
+    private static final String CLIENT_CREDENTIALS = "client_credentials";
+    private static final String CLIENT_ID = "test-client";
+    private static final String SCOPE_READ_WRITE = "READ_WRITE";
+    private static final String MEANS_ASSESSMENT_ENDPOINT_URL = "/api/internal/v1/assessment/means/stateless";
+
+    private MockMvc mvc;
+
+    @Autowired
+    private WebApplicationContext webApplicationContext;
+
+    @Autowired
+    private FilterChainProxy springSecurityFilterChain;
+
+    @Before
+    public void setup() {
+        this.mvc = MockMvcBuilders.webAppContextSetup(webApplicationContext)
+                .addFilter(springSecurityFilterChain)
+                .build();
+    }
+
+    @Test
+    public void validJsonProducesSuccessResult() throws Exception {
+        var income = Map.of("income_type", IncomeType.EMPLOYMENT_INCOME.getValue(),
+                "applicant", Map.of("frequency", Frequency.ANNUALLY.name(),
+                        "amount", BigDecimal.valueOf(2700)),
+                "partner", Map.of("frequency", Frequency.ANNUALLY.name(),
+                        "amount", BigDecimal.valueOf(2700)));
+        var tax = new Outgoing(OutgoingType.TAX,
+                new FrequencyAmount(Frequency.ANNUALLY, BigDecimal.valueOf(1700)),
+                new FrequencyAmount(Frequency.ANNUALLY, BigDecimal.valueOf(1700))
+        );
+        var ni = new Outgoing(OutgoingType.NATIONAL_INSURANCE,
+                new FrequencyAmount(Frequency.ANNUALLY, BigDecimal.valueOf(700)),
+                new FrequencyAmount(Frequency.ANNUALLY, BigDecimal.valueOf(700))
+        );
+        var outgoings = Stream.of(tax, ni).map(x -> outgoingToJson(x)).toList();
+
+        var assessment = Map.of("assessment_date", "2023-05-02T00:00:00",
+                "assessment_type", "INITIAL",
+                "case_type", "EITHER WAY",
+                "magistrates_court_outcome", "COMMITTED FOR TRIAL",
+                "has_partner", false,
+                "dependant_children", Arrays.asList(
+                        Map.of("age_range", "0-1", "count", 2),
+                        Map.of("age_range", "5-7", "count", 1)
+                ));
```

Comment:
In most of the other tests we're using Jackson to map Java object to JSON strings. I feel like that is a much more simple and flexible approach that we should also be using here:

```
@Autowired
private ObjectMapper objectMapper;

var assessment = new StatelessAPIRequest()
    .withAssessmetnType(StatelessRequestType.INITIAL)
    .withCaseType(CaseType.EITHER_WAY)
    .withHasPartner(false)

String json = objectMapper.writeValueAsString(assessment);
```